### PR TITLE
Remove file create time.

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Because our prod file systems don't support birth date.

## Was the usage documentation (e.g. openapi.yml, README, DevOpsDocs) updated?
Yes.

## Does this change affect how this application integrates with other services?
No.

If so, please confirm:
- [ ] change was tested on stage    and/or
- [ ] test added to sul-dlss/infrastructure-integration-test

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,11 +10,23 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_02_26_055313) do
+ActiveRecord::Schema.define(version: 2020_02_27_211823) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "dro_file_parts", force: :cascade do |t|
```
Comment (mjgiarlo) [inline db/schema.rb]: Is this related to this PR?

Comment (justinlittman) [reply db/schema.rb]: No. Let me fix that.

Comment (justinlittman) [reply db/schema.rb]: Fixed.
